cwms-data-api/src/main/java/cwms/cda/security/OpenIDAccessManager.java

```diff
@@ -42,6 +43,8 @@
 public class OpenIDAccessManager extends CdaAccessManager {
     private static final FluentLogger log = FluentLogger.forEnclosingClass();
     public static final String AUTHORIZATION = "Authorization";
+    public static final String CREATE_USERS_KEY = "cda.openid.create_users";
+    private static final boolean createUsers = true;// Boolean.parseBoolean(System.getProperty(CREATE_USERS_KEY,"false"));
```

Comment:
I'd say either leave the property in but default to true or removed the commented out property bit